packages/exoflex/package.json

```diff
@@ -34,12 +35,13 @@
     "lodash.clamp": "4.0.3",
     "lodash.mergewith": "4.6.2",
     "lodash.tonumber": "4.0.3",
+    "patch-package": "^6.5.1",
     "react-native-animation-hooks": "^1.0.1",
-    "react-native-calendars": "https://github.com/oshimayoan/react-native-calendars/archive/oshimayoan-0.0.3.tar.gz",
-    "react-native-collapsible": "^1.5.1",
+    "react-native-calendars": "^1.1293.0",
```

Comment:
On reverting this to the newest feat, have we test these change support what Yoan originally patch out?

Reply:
I think it's already supported on the newest version, but let me include @JonathanRadotski to the discussion. What do you think?

Reply:
Briefly checked and comparing added feature. I think it is supported with added feature from previous package and good to go 👍.